packages/firebase_data_connect/firebase_data_connect/example/lib/generated/delete_movie.dart

```diff
@@ -0,0 +1,89 @@
+part of movies;
+
+class DeleteMovie {
+  String name = "deleteMovie";
+  DeleteMovie({required this.dataConnect});
+
+  Deserializer<DeleteMovieResponse> dataDeserializer = (String json) =>
+      DeleteMovieResponse.fromJson(jsonDecode(json) as Map<String, dynamic>);
+  Serializer<DeleteMovieVariables> varsSerializer =
+      (DeleteMovieVariables vars) => jsonEncode(vars.toJson());
+  MutationRef<DeleteMovieResponse, DeleteMovieVariables> ref(
+      {required String id, DeleteMovieVariables? deleteMovieVariables}) {
+    DeleteMovieVariables vars1 = DeleteMovieVariables(
+      id: id,
+    );
+    DeleteMovieVariables vars = deleteMovieVariables ?? vars1;
+    return dataConnect.mutation(
+        this.name, dataDeserializer, varsSerializer, vars);
+  }
+
+  FirebaseDataConnect dataConnect;
+}
+
+class DeleteMovieMovieDelete {
+  late String id;
+
+  DeleteMovieMovieDelete.fromJson(Map<String, dynamic> json)
+      : id = json['id'] {}
+
+  // TODO(mtewani): Fix up to create a map on the fly
```

Comment:
Are these todos relevant btw?

Reply:
These todos are generated when using the VSCode extension, so nothing we can directly do in this repository.